### PR TITLE
Update saihostintf.h

### PR DESCRIPTION
**Pull request by tushar-ty**
Add outbound cos queue number selection attribute.

### CODE REVIEW

File: inc/saihostintf.h [hunk 1]
```diff
@@ -526,6 +526,9 @@ typedef enum _sai_hostif_attr_t
 
     /** Set the operational status for this host interface [bool] (default to false) */
     SAI_HOSTIF_ATTR_OPER_STATUS,
+    
+    /** Set the cos queue number to be used for packets going out through this interface */
+    SAI_HOSTIF_ATTR_COS_QUEUE,
```
Comment (kcudnik) [inline inc/saihostintf.h]: Should that be queue index ? or actual QUEUE object ID? Please provide value type.

Comment (tushar-ty) [inline inc/saihostintf.h]: Yes, its an index not an object. Will update.

File: inc/saihostintf.h [hunk 2]
```diff
@@ -526,6 +526,10 @@ typedef enum _sai_hostif_attr_t
 
     /** Set the operational status for this host interface [bool] (default to false) */
     SAI_HOSTIF_ATTR_OPER_STATUS,
+    
+    /** Set the cos queue number to be used for packets going out through this interface [uint32_t] 
+    * (default 0) This is an index. */
+    SAI_HOSTIF_ATTR_COS_QUEUE,
```
Comment (SathishKumarTR-Dell) [inline inc/saihostintf.h]: Will this mean the same cos queue is used for all packets? There is a new attribute SAI_PACKET_EGRESS_QUEUE proposed in the Host IF enhancements proposal (SAI-Proposal-4-host-if-ver7.docx) which would solve this requirement?

**Comment by lguohan** [comment]
test

**Comment by lguohan** [comment]
@tushar-ty , the build fails due to missing meta data for this attribute. Can you help add the meta data for this attribute?

**Pull request closed without merging**